src/main/java/frc/robot/util/CompoundCommands.java

```diff
@@ -277,15 +299,15 @@ public static DeferredCommand deferDrive(Supplier<Command> command) {
   public static Command armToGoal(double wristSetPos, double pivotSetPos, double elevatorSetPos) {
```

Comment:
I feel like the "moving wrist to safe position" should be a part of this armToGoal command and not separate. Not sure though, and I think better suited for a later refactor, since you have armToReefSafely and armToStowSafely already.